app/uk/gov/hmrc/helptosavestub/controllers/NSIController.scala

```diff
@@ -124,10 +124,20 @@ object NSIController extends BaseController with Logging {
           BadRequest(Json.toJson(NSIErrorResponse(version, correlationId, errors.toList)))
         }, {
           validatedNino ⇒
-            if (validatedNino.contains("401")) {
+            if (validatedNino.contains("400")) {
+              BadRequest(Json.toJson(NSIErrorResponse.missingNinoError))
+            } else if (validatedNino.contains("401")) {
               Unauthorized
+            } else if (validatedNino.contains("404")) {
```

Comment:
some of these status codes aren't in the ICD, what's the reason for wanting the stub to return codes that aren't in the spec of the real API?